.github/workflows/lock.yml

```diff
@@ -0,0 +1,34 @@
+name: 'Lock old threads'
+
+on:
+  schedule:
+    - cron: '0 0 * * *'
+
+permissions:
+  issues: write
+  pull-requests: write
+
+jobs:
+  action:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: dessant/lock-threads@v2
+        with:
+          issue-lock-inactive-days: '30'
+          pr-lock-inactive-days: '30'
```

Comment:
Is it possible to only lock threads that contain the `Support` label?

Reply:
Nop, we can only exclude certain labels using `issue-exclude-labels` https://github.com/dessant/lock-threads/#inputs